.circleci/config.yml

```diff
@@ -383,3 +391,20 @@ workflows:
           <<: *default-context
           react-version: next
           name: test_e2e-react@next
+
+  material-ui-v6:
+    when:
+      equal: [material-ui-v6, << pipeline.parameters.workflow >>]
+    jobs:
+      - test_unit:
+          <<: *default-context
+          name: test_unit-material@next
+      - test_browser:
+          <<: *default-context
+          name: test_browser-material@next
+      - test_regressions:
+          <<: *default-context
+          name: test_regressions-material@next
+      - test_e2e:
+          <<: *default-context
+          name: test_e2e-material@next
```

Comment:
I think I'll add the `test_types` job here as well.

Reply:
That makes a lot of sense. 👍

Reply:
Added